### ISSUE TITLE
[pmts-mlc] "DIMM power budget not supported" reported with additional script errors

### ISSUE THREAD

**Issue by sscargal** [sscargal/pmts#2]
When running pmts-mlc on Ubuntu, we get the following errors:

```
$ sudo ./pmts-mlc.sh -m /usr/local/bin/mlc -s0 -p /aep/02
=======================================================================
Starting Intel PMem bandwidth and latency measurements using MLC
pmts-mlc.sh Version 1.0.0
Test Started: Wed Apr 29 07:09:09 PDT 2020
=======================================================================
Using MLC command: /usr/local/bin/mlc
MLC version: v3.8
Using MLC AVX512: Yes
Using NDCTL command: /bin/ndctl
NDCTL version: 64.1
Using IPMCTL command: /bin/ipmctl
IPMCTL version: 01.00.00.4104
Test results: ./mlc-outputs.0429-0709
Using pmem file system: /aep/02
/aep/02 file system type: ext4
Operating System: CentOS Linux 7 (Core)
CPU cores per socket: 20
CPUs on Socket 0:      0-19,40-59
PMem DIMM Health: Healthy
PMem DIMM Capacity: 128GiB
ARS Status: Completed
./pmts-mlc.sh: line 416: ((: > 9500 : syntax error: operand expected (error token is "> 9500 ")
./pmts-mlc.sh: line 418: ((: > 11501 : syntax error: operand expected (error token is "> 11501 ")
./pmts-mlc.sh: line 420: ((: > 14501 : syntax error: operand expected (error token is "> 14501 ")
./pmts-mlc.sh: line 422: ((: > 17501 : syntax error: operand expected (error token is "> 17501 ")
DIMM power budget not supported
--- 8< ---
```

**Comment by sscargal** [comment]
The root cause is we do not get `AvgPowerBudget` reported on old PMem firmware, so the following grep returns a NULL dataset.

```
+ DIMMS_POWER_BUDGET=($(${GREP} -w "AvgPowerBudget" "${OUTPUT_PATH}/dimm_info.dat" | cut -d'=' -f 2 | ${AWK} '{ print $1}'))
++ /bin/grep -w AvgPowerBudget ./mlc-outputs.0429-0759/dimm_info.dat
```

We need to add a check for `DIMMS_POWER_BUDGET` to determine if it is NULL or not. Since this is purely informational, it is a non-fatal error if it is NULL and we should continue.

**Comment by sscargal** [comment]
With the suggested fix in place, we now report `Not Available` instead of the error.

```
PMem DIMM AvgPowerBudget: Not Available
```

**Comment by sscargal** [comment]
Resolved with #5
